### PR TITLE
fix: Fix broken camera push.

### PR DESCRIPTION
**Pull request by balloonpopper**
Fixes push to work as per :
https://docs.escoria-framework.org/en/devel/scripting/z_esc_reference.html#camera-push-target-time-type-api-doc
Fixes : https://github.com/godot-escoria/escoria-issues/issues/125

### CODE REVIEW

File: addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd [hunk 1]
```diff
@@ -4,11 +4,16 @@
...
+#
 # **Parameters**
 #
-# - *target*: Global ID of the `ESCItem` to push the camera to. If the target
-#   has a child node called `camera_node`, its location will be used. If not,
-#   the location of the target will be used
+# - *target*: Global ID of the `ESCItem` to push the camera to. `ESCItem`s have
+#   a "camera_node" property that can be set to point to a node (usually an
+#   `ESCLocation` node). If the "camera_node" property is empty, `camera_push`
+#	will point the camera at the `ESCItem`s location. If however, the `ESCItem`
```
Comment (dploeger) [inline addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd:14]: Remove the extra whitespace here and in line 16.

**Review by dploeger — CHANGES_REQUESTED** [review]

File: addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd (hunk 1 above)
Comment (balloonpopper) [reply addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd]: Fixed and rebased.

**Review by dploeger — APPROVED** [review]
Great. Thanks. 🎁

**Review by StraToN — APPROVED** [review]

**Comment by StraToN** [comment]
Thanks !

**Pull request merged**